src/settings/settingsdialog.cpp

```diff
@@ -120,6 +121,7 @@ SettingsDialog::SettingsDialog(Application *app, QWidget *parent)
   QTreeWidgetItem *general = AddCategory(tr("General"));
   AddPage(Page_Behaviour, new BehaviourSettingsPage(this), general);
   AddPage(Page_Collection, new CollectionSettingsPage(this), general);
+  AddPage(Page_Context, new ContextSettingsPage(this), general);
```

Comment:
I think we want to place this in the user interface section instead between appearance and notifications.